src/main/resources/i18n/Messages_en.properties

```diff
@@ -119,13 +119,15 @@ manifestCsvVariableNameNotUniqueWithinParent=No two children of the same parent
 manifestCsvVariableParentDoesNotExist=Parent for variable does not exist
 manifestCsvWrongDataTypeForChild=Only tables and sections may be listed as parents
 # {0} is plot name
-monitoringPlotNortheastCorner={0} Northeast Corner
+monitoringPlotNortheastCorner={0} NE - Plot type: {1} - Planting Zone Name: {2} - Planting Subzone Name: {3}
 # {0} is plot name
-monitoringPlotNorthwestCorner={0} Northwest Corner
+monitoringPlotNorthwestCorner={0} NW - Plot type: {1} - Planting Zone Name: {2} - Planting Subzone Name: {3}
 # {0} is plot name
-monitoringPlotSoutheastCorner={0} Southeast Corner
+monitoringPlotSoutheastCorner={0} SE - Plot type: {1} - Planting Zone Name: {2} - Planting Subzone Name: {3}
 # {0} is plot name
-monitoringPlotSouthwestCorner={0} Southwest Corner
+monitoringPlotSouthwestCorner={0} SW Corner - Plot type: {1} - Planting Zone Name: {2} - Planting Subzone Name: {3}
```

Comment:
Just looked at the Jira task and it looks like the SW corner string should have the word "Plot" at the beginning. (But not the other corners.)